tests/fixtures/integration/actions/templar/test_direct_interactive_noee.py/test/5.json

```diff
@@ -0,0 +1,88 @@
+{
+    "name": "test[5-:doc-doc for task]",
+    "index": 5,
+    "comment": "doc for task",
+    "additional_information": {
+        "look_fors": [
+            "module: debug"
+        ],
+        "look_nots": [],
+        "compared_fixture": false
+    },
+    "output": [
+        "DEBUG (MODULE)",
```

Comment:
Can we use `look_fors` for check if module doc is displayed. Having a entire module doc in fixture can result in failure if the actual debug module doc changes in future ansible version?